logback/src/main/resources/logback.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<configuration>
+
+  <appender name="STDOUT" class="ch.qos.logback.core.ConsoleAppender">
+    <encoder>
+      <Pattern>%d{YYY-dd-MM HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</Pattern>
+    </encoder>
+  </appender>
+   <appender name="loki" class="com.grafana.loki.LokiAppender">
+    <encoder>
+      <Pattern>%d{YYY-dd-MM HH:mm:ss.SSS} [%thread] %-5level %logger{36} - %msg%n</Pattern>
+    </encoder>
+    <lokiConfig>
+    	<url>http://localhost:3100</url>
+    	<tenantID>fake</tenantID>
```

Comment:
Can you include 
```suggestion
    	<tenantID>fake</tenantID> <!-- optional -->
```

And this should default to not sending the header if not supplied

Reply:
nevermind looks like it's already the case you don't send it if this is not supplied, I still think we should include the optional annotation as this will be the reference file everyone uses to configure this